atlp-rwanda/Team-Sostene-E-commerce-fe
# #185172093 add a product to the created collection in data base, delete collection, and validate create collection

#### What does this PR do?
- Enables a seller to add a product with images to the database
- Enables a seller to delete a collection created before, with erase of all products
- a seller can not create nullable or empty string collection name

#### Description of Task to be completed?
- used formik  and Yup for form validation
- by uploading images , a seller can use drag and drop feature or upload one manually
- etc

#### How should this be manually tested?

Via hosted link, you can log in as a seller, navigate to the dashboard(right navbar), and then you will see the list of collections, then you can add product by clicking on add button, or clicking on collection down.

#### Any background context you want to provide?
none
#### What are the relevant pivotal tracker/Trello stories?
#185172093
#### Screenshots (if appropriate)
--- testable via the link
#### Questions:
